spec/workspace-spec.js

```diff
@@ -2528,6 +2528,62 @@ i = /test/; #FIXME\
     })
   })
 
+  describe('::activateNextPane', () => {
+    describe('when the active workspace pane is inside a dock', () => {
```

Comment:
Thanks, @nathansobo.

> I think you could just say "pane" instead of "workspace pane"

I thought about that. I opted for the more verbose language ("workspace pane") to clarify that  we're talking about the workspace's active pane, as opposed to the active pane inside a specific pane container like the dock or workspace center. IIRC, each dock and the workspace center will have an active pane, but only one of those panes will be the workspace's active pane.